opm/parser/eclipse/EclipseState/Schedule/Well/WList.hpp

```diff
@@ -28,29 +28,33 @@ namespace Opm {
 
 class WList {
 public:
-    using storage = std::unordered_set<std::string>;
+    using storage = std::vector<std::string>;
 
     WList() = default;
     WList(const storage& wlist);
+    std::size_t insert_index() const;
+    std::string name() const;
     std::size_t size() const;
+    void setName(const std::string& wListName);
```

Comment:
Instead of `setName()`the name should be supplied as a constructor argument.

Reply:
OK - changed